pkgs/tools/security/hakrawler/default.nix

```diff
@@ -0,0 +1,33 @@
+{ buildGoModule
+, fetchFromGitHub
+, stdenv
+}:
+
+buildGoModule rec {
+  pname = "hakrawler";
+  version = "20201224-${stdenv.lib.strings.substring 0 7 rev}";
+  rev = "e39a514d0e179d33362ee244c017fb65cc2c12a5";
+
+  src = fetchFromGitHub {
+    owner = "hakluke";
+    repo = "hakrawler";
+    inherit rev;
+    sha256 = "0wpqfbpgnr94q5n7i4zh806k8n0phyg0ncnz43hqh4bbdh7l1y8a";
+  };
+
+  vendorSha256 = "18zs2l77ds0a3wxfqcd91h269g0agnwhginrx3j6gj30dbfls8a1";
+
+  doCheck = true;
```

Comment:
```suggestion
```

This is true by default.

Reply:
Thanks for your feedback.

Removed

Reply:
I think you did something wrong when force pushing because the line is back.

Reply:
Yes, it did. Not saving the file and I can force push all day long with no effect :wink: